### PR TITLE
[Bugfix] Minor bug fixes

### PR DESCRIPTION
**Pull request by Edw1x**
# Bugfix/#2737 search items should have frames
### Result : Added borders 
image

# Bugfix/#2707 Wrong size of 'Eco news' link, Twitter icon and 'Follow us' label with icons of social networks on the footer
### Result : sizes of elements changed
image
image

# Bugfix/#2745 [My habits] "My habits" should be renamed 
### Result : renamed
image

# Bugfix/#2721 [Search] Long search result doesn't move on the next line when user types long value
### Result: long search was cropped
image

# Bugfix/#2716 [Search] The search window is closed after the user clicks on any page place
### Result: no redirection when the user clicks on the empty fields 

# Bugix/#2701 [My habits] Number of 'My achievements' and 'My eco-friends' are displayed incorrect.
### Result: numbers displayed correct
image

# Bugfix/#2711 [Edit profile] Possible to save name with a dot in the 'Name' field
### Result: no possibility to save name with dot
image

# Bugfix/#2714 Wrong quantity of news are displayed for screen resolution 1024
### Result: correct quantity
image

### CODE REVIEW

File: src/app/main/component/layout/components/search-popup/search-popup.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
   </div>
   <!-- search results -->
   <div class="search_body-wrapper">
-    <div class="search-content-wrapper container" (click)="closeSearch()">
```
Comment (JimHolman1199) [inline src/app/main/component/layout/components/search-popup/search-popup.component.html:26]: why do we deleted this handler ?

Comment (JimHolman1199) [reply src/app/main/component/layout/components/search-popup/search-popup.component.html:26]: ah.. got it

File: src/app/main/component/layout/components/footer/footer.component.scss [hunk 2]
```diff
@@ -56,7 +56,12 @@ li {
 .footer_social-link {
   width: 16px;
   height: 16px;
-  margin: 0 7px;
+  margin: 0 6px;
+
+  img {
+    width: 15px;
+    height: 12.19px;
```
Comment (JimHolman1199) [inline src/app/main/component/layout/components/footer/footer.component.scss:63]: why 12.19px ?

Comment (Edw1x) [reply src/app/main/component/layout/components/footer/footer.component.scss:63]: According to mockups:
   The size of the Twitter icon is 15x12.19

¯\_(ツ)_/¯

**Review by paslavskyi9roman — APPROVED** [review]

**Review by SofiiaBaziv — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

**Comment by JimHolman1199** [comment]
fix conflict so we can merge it

**Review by JimHolman1199 — APPROVED** [review]

**Pull request merged**